import-export-cli/specs/v2/swagger2.go

```diff
@@ -216,6 +217,14 @@ func Swagger2Populate(def *APIDefinition, document *loads.Document) error {
 	if basepath, ok := swagger2XWO2BasePath(document); ok {
 		def.Context = path.Clean(basepath)
 		def.ContextTemplate = path.Clean(basepath)
+		if !strings.Contains(basepath, "{version}") {
+			def.Context = path.Clean(basepath + "/" + def.ID.Version)
```

Comment:
If base path already ends with /, will this add double slashes.
for ex if basepath is /foo/bar/ then it will set context as foo/bar//$VERSION

Reply:
Here I have used path.Clean to remove such extra slashes as shown below. 
![image](https://user-images.githubusercontent.com/25246848/77422934-509aee00-6df4-11ea-984a-e9779d6c3297.png)